03_Uitvoering.Rmd

```diff
@@ -60,14 +59,13 @@ check <- raster_meetnet_poly %>% st_drop_geometry() %>% count(rasterid) %>% filt
 ## Selectie rastercellen
 
 Op basis van deze rasterlaag maken we een overlay met de verspreiding van de vijf grondwatergroepen volgens de habitatkaart (figuur \@ref(fig:overlay-habitatkaart-GRTS-plot)). 
-
+We deden hiervoor een beroep op de gestandaardiseerde versie van de habitatkaart ([bron](https://zenodo.org/record/3355193)). 
```

Comment:
Best refereren als een referentie met DOI, zoals het ook op Zenodo te zien is (en als BibTeX kan worden getoond/gedownload). Zoals het ook voor habitatmap_terr is gebeurd.

Reply:
Je URL refereert nog naar de 1e versie (2018_v1); ik weet niet welke je precies hebt gebruikt, maar n2khab verwacht default versie 2 (2018_v2) dus het zal die wel moeten zijn denk ik. Maar gebruik steeds een DOI-link: https://doi.org/....

Reply:
OK, heb de link naar versie 2 gegeven